### PR TITLE
docs: update synchronization status check docs

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendu@apache.org>

Updates the tutorial on checking the synchronization status of APISIX CRDs.

### CODE REVIEW

File: docs/en/latest/tutorials/check-crd-status.md [hunk 1]
```diff
@@ -21,28 +27,17 @@ title: How to quickly check the synchronization status of CRD
...
-When the Apache APISIX Ingress Controller watches the resource change, the logic unit of the Apache APISIX Ingress Controller has just started to work.
-In various operations of the Apache APISIX Ingress Controller, object conversion and more verification will be performed.
-When the verification fails, the Apache APISIX Ingress Controller will log an error message and will continue to retry until the declared state is successfully synchronized to APISIX.
-
-Therefore, after the declarative configuration is accepted by Kubernetes, it does not mean that the configuration is synchronized to APISIX.
-
-In this practice, we will show how to check the status of CRD.
-
-## Prerequisites
+APISIX CRDs are applied to a Kubernetes cluster using the `kubectl apply command`. Behind the scenes, Kubernetes verifies the configuration using the [Open API V3 schema](https://swagger.io/specification/) and its validation webhooks (if any).
```
Comment (SylviaBABY) [inline docs/en/latest/tutorials/check-crd-status.md:30]:
```suggestion
APISIX CRDs are applied to a Kubernetes cluster using the `kubectl apply` command. Behind the scenes, Kubernetes verifies the configuration using the [Open API V3 schema](https://swagger.io/specification/) and its validation webhooks (if any).
```

Comment (navendu-pottekkat) [reply docs/en/latest/tutorials/check-crd-status.md]: Missed it, thanks.

**Review by SylviaBABY — APPROVED** [review]
LGTM

File: docs/en/latest/tutorials/check-crd-status.md [hunk 2]
```diff
@@ -21,28 +27,17 @@ title: How to quickly check the synchronization status of CRD
...
-When the Apache APISIX Ingress Controller watches the resource change, the logic unit of the Apache APISIX Ingress Controller has just started to work.
-In various operations of the Apache APISIX Ingress Controller, object conversion and more verification will be performed.
-When the verification fails, the Apache APISIX Ingress Controller will log an error message and will continue to retry until the declared state is successfully synchronized to APISIX.
-
-Therefore, after the declarative configuration is accepted by Kubernetes, it does not mean that the configuration is synchronized to APISIX.
-
-In this practice, we will show how to check the status of CRD.
-
-## Prerequisites
+APISIX CRDs are applied to a Kubernetes cluster using the `kubectl apply` command. Behind the scenes, Kubernetes verifies the configuration using the [Open API V3 schema](https://swagger.io/specification/) and its validation webhooks (if any).
```
Comment (juzhiyuan) [inline docs/en/latest/tutorials/check-crd-status.md:30]:
```suggestion
APISIX CRDs are applied to a Kubernetes cluster using the `kubectl apply` command. Behind the scenes, Kubernetes verifies the configuration using the [OpenAPI V3 schema](https://swagger.io/specification/) and its validation webhooks (if any).
```

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**